convert2rhel/utils.py

```diff
@@ -342,6 +342,12 @@ def is_dir_empty(path):
 def remove_pkgs(pkgs_to_remove, backup=True, critical=True):
     """Remove packages not heeding to their dependencies."""
     loggerinst = logging.getLogger(__name__)
+    if backup:
```

Comment:
@SpyTec, FYI, I moved this back to the top of the function because if left at the bottom it's impossible to backup the packages. The removal of centos/oraclelinux-release pkg causes that the original repos are not available. And they need to be available for downloading the pkgs for backup.